_posts/2020-07-10-removing-negative-words

```diff
@@ -0,0 +1,86 @@
+
+As such, it is important to have the discussion around the necessity of this change, it's motivation and it's impact. Of course large corporations can simply get their executives to issue a top-bottom order forbidding the use of these terms. But it might prove difficult to convince independent developers and smaller projects to adopt the new terms, and quickly.
+
+Avoiding the discussion does very little to raise awareness to and promote the actual understanding of the issue. It is important to explain the arguments behind it and address the concerns around it.
+
+In an effort to do that, I'll talk a bit more about motivation and the arguments supporting the change.
+
+## Motivation
+
+As previously mentioned, this is far from a recent issue. It is, however, seeing more and more adoption lately. And more questioning.
+
+The motivation to move away from "whitelist / blacklist" is pretty clear. The use of black and white in this context has nothing to do with color or light, and everything to do with the perception that black is bad and white is good, which is deeply rooted in racism. There is no good reason to use a metaphor when there are clear, descriptive terms to be used instead. Let alone use a racially charged one.
+
+Moving away from the "master / slave" terminology, however, has seen more controversy. As such, the following sections will be dedicated to that particular change.
+
+As Matthew Ahrens eloquently explains in his [pull request](https://github.com/openzfs/zfs/pull/10435) removing all possible references to the term "slave" from OpenZFS' `zfs` repository:
+
+>The horrible effects of human slavery continue to impact society. The casual use of the term "slave" in computer software is an unnecessary reference to a painful human experience.
+
+Empathy in itself is a pretty good motivation for change and knowing that these terms are hurtful and can easily be replaced are sufficient to prompt us to act. However, we also believe in understanding why we are doing things and how what we are doing fits with our company culture and purpose. Simply following the trend without questioning why it's important, how it's beneficial and where it fits in a bigger picture context isn't enough.
+
+Understanding these points is paramount to create a culture of ambassadors of change. Questioning, done with an open mind and actual desire to understand, is healthy and desirable.
+
+### Why is a change in language needed?
+
+There is an intrinsic relationship between language and power. And there is an unstopable evolution in language. Every day, new terms arise and old terms become obsolete.
+
+The "master / slave" terminology is clearly connected to slavery. And slavery is far from being a concept "lost in time". First and foremost because the issue of human slavery should never be forgotten. And second because slavely is, sadly, far from being an issue that is no longer present in society.
```

Comment:
```suggestion
The "master / slave" terminology is clearly connected to slavery. And slavery is far from being a concept "lost in time". First and foremost because the issue of human slavery should never be forgotten. And second because slavery is, sadly, far from being an issue that is no longer present in society.
```